starter-code/scripts/articleView.js

```diff
@@ -37,15 +41,26 @@ articleView.handleCategoryFilter = function() {
   /* TODO: Just like we do for #author-filter above, we should also handle
   change events on the #category-filter element. Be sure to reset the
   #author-filter while you're at it! */
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      $(`article[data-category="${$(this).val()}"]`).fadeIn(500);
+    } else {
+      $('article').show();
```

Comment:
As with above, try to be more specific in your selectors. Another point - this will also cause your article template to show, which isn't really what you want. Instead, you could try this:
```
$('#articles article').show();
$('#articles article.template').hide();
```